src/app.ts

```diff
@@ -599,9 +650,75 @@ function fetchDescriptionGenerationData(
       ),
     }).predict(prompt)
   );
+  const descriptionKeywordPrefix =
```

Comment:
why isn't this integrated into the "predict" method? I think it's cleaner to have it as `const res`, so that "predict" covers all use cases (title generation, description generation, description evaluation).